.devcontainer/devcontainer.json

```diff
@@ -1,13 +1,11 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
 {
-	"name": "Ubuntu",	
-	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile	
-	"image": "mcr.microsoft.com/devcontainers/base:jammy",
-	"features": {
-		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {}
-	},
-
+	"name": "Docker in Docker",
```

Comment:
Wonder if you want to name is in context of our solution? e.g. "Wasm Orchestration With Spin"